deltaq/BsDiff/BsDiff.cs

```diff
@@ -38,7 +37,7 @@ public static class BsDiff
         internal const int HeaderSize = 32;
         internal const long Signature = 0x3034464649445342; //"BSDIFF40"
 
-        private static readonly Lazy<ISuffixSort> DefaultSuffixSort = new Lazy<ISuffixSort>(() => new SAIS());
+        private static readonly ISuffixSort DefaultSuffixSort = new SAIS();
```

Comment:
This is incredibly tiny but I figured I'd still mention why I made this change. `SAIS()` contains no field or init code and literally just exists to be an interface impl, so constructing it is *literally* cheaper than creating the `Lazy<T>` instance. Except even then the `Lazy<T>` doesn't make sense because this static field is implicitly its own `Lazy<T>` due to how `beforefieldinit` works in the CLR.